services/net/contentmigration/sources/oracle/Entities/NewsItem.cs

```diff
@@ -329,4 +329,10 @@ public class NewsItem
     /// </summary>
     [Column("EOD_DATE")]
     public string? EodDateTime { get; set; }
+
+    /// <summary>
+    /// get/set
+    /// </summary>
+    public ICollection<UserTone>? Tones { get;} = new List<UserTone>();
```

Comment:
Normally relationships are not nullable, especially if you initialize them.

Reply:
fixed